config/rbac/role.yaml

```diff
@@ -86,6 +86,24 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
+  - metal3.io
+  resources:
+  - hostdata
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - metal3.io
+  resources:
+  - hostdata/finalizers
```

Comment:
This is new to me, why do we need to add `finalizers` in rbac? also, do not you need to add `hostdata/status` in this list?

Reply:
I don't know how status ended up here, but it there is no status in the CRD, probably it is a leftoover from the clean up.
hostdata/finalizers is necessary to update finalizers on hostData upon deletion